src/utils/api.js

```diff
@@ -5,10 +5,42 @@ const newslyApi = axios.create({
 })
 
 export const getArticles = () => {
-    return newslyApi.get('/articles')
-        .then((res) => {
-            console.log(res.data.articles, "<<<<<<< from api")
-            return res.data.articles;
-            
+    return newslyApi
+        .get('/articles')
+        .then(({data: {articles}}) => {
+            return articles;
         })
-}
\ No newline at end of file
+}
+
+export const getArticle = (article_id) => {
+    return newslyApi
+        .get(`/articles/${article_id}`)
+        .then(({data: {article}}) => {
+            return article;
+    })
+}
+
+// export const deleteArticle = (article_id) => {
+//     return newslyApi.delete(`/articles/${article_id}`).then(() => {
+//         return [];
+//     })
+// }
+
+// export const addArticle = (form) => {
+//     return newslyApi
+//     .post("/articles", form)
+//     .then((res) => {
+//         return res.data.article.article_id;
+//     })
+// }
+
+// export const getTopics = () => {
+//     return newslyApi.get("/topics").then((res) => {
+//         return res.data.topics;
+//     })
+// }
+
+// export const addTopic = (form) => {
+//     return newslyApi
+//     .post("/topics", form)
```

Comment:
why are all of these in here? Make sure you working through the project in vertical slices.

Reply:
Removed.